test-data/unit/fine-grained.test

```diff
@@ -1327,9 +1327,9 @@ def g() -> None: pass
 def g() -> int: pass
 [builtins fixtures/dict.pyi]
 [out]
-main:5: error: Need type annotation for variable
+main:5: error: Need type annotation for 'x'
 ==
-main:5: error: Need type annotation for variable
+main:5: error: Need type annotation for 'x'
```

Comment:
I think you should also add dedicated tests for more tricky situations like:
```python
x, y = [], []
self.a, self.b = None, None
```
(and maybe some others if you have more ideas).

Reply:
I added a case for multiple assignment. I think classes would be a valid case too but currently mypy doesn't appear to require annotations for `self` assignments. Other cases I can't think of. My understanding is the only assignments that can trigger this are assignments with variable names literal as lvalue, which is covered.